packages/babel-parser/src/parser/error-message.js

```diff
@@ -4,8 +4,8 @@
 // The Errors key follows https://cs.chromium.org/chromium/src/v8/src/common/message-template.h unless it does not exist
 export const ErrorMessages = Object.freeze({
   AccessorIsGenerator: "A %0ter cannot be a generator",
-  ArgumentsDisallowedInInitializer:
-    "'arguments' is not allowed in class field initializer",
+  ArgumentsInClass:
```

Comment:
Is it confusing to mention `or static block` if a user doesn't have it enabled (should we only show it if they do?)?

Reply:
I try to avoid different message per plugins because they are hard to maintain. 😄 What about:
```
'arguments' is only allowed in functions and class methods.
```

Reply:
👍